packages/typespec-ts/test/integration/generated/models/propertyAdditional/src/models.ts

```diff
@@ -124,3 +261,5 @@ export type ExtendsUnknownAdditionalPropertiesDiscriminated =
 export type IsUnknownAdditionalPropertiesDiscriminated =
   | IsUnknownAdditionalPropertiesDiscriminatedParent
   | IsUnknownAdditionalPropertiesDiscriminatedDerived;
+/** Alias for WidgetData */
+export type WidgetData = WidgetData0 | WidgetData1;
```

Comment:
Not sure we should not include this model considering it is not referenced?

Reply:
currently our clear unused models function didn't consider types, I found this in https://github.com/Azure/autorest.typescript/pull/2432 too.